### DISCUSSION TITLE
Any plan on supporting backlinks?

### DISCUSSION THREAD

**Discussion by yuchanns** [vuepress-theme-hope/vuepress-theme-hope#1798] [Q&A] [answered]
Backlinks, aka Bi-directional links, or zettelkasten, is useful for personal knowledge management. A benefit is allow the writer to indicate the importance of a potentially interesting or connecting topic without breaking concentration, and write on a topic which has content associated to it later. Having those easily at your fingertips and collected in the background should help build the content and associated topics which writer may have forgotten or didn't think about the first time around.

There are some other products have implemented this feature, such as Roam Search, Obsidian, Logseq, Notion and so on. I do find a [vuepress plugin](https://github.com/gcoda/vuepress-backlinks) also supported it but stopped on v1 many years ago.

**Comment by Mister-Hope** [comment, marked as answer]
I am not finding this useful. Any detailed explaination why is it different to a normal link?

If it’s just some special link syntax, for some specific markdown notes software to generate a link map, it’s definitely not something VuePress is based on and should be interested in.

More over, vuepress will extract any links in page and put them to env, and that’s how we support “Dead Link Check” in `VuePress-plugin-md-enhance`. If you want a link map, you may implement one using that. So unless further more convincing reason, I will say No to this feature.

**Reply by Mister-Hope** [reply]
I just don't know what's the difference with a normal link. Any explanation?

**Reply by yuchanns** [reply]
> I just don't know what's the difference with a normal link. Any explanation?

In my mind, the major distinction is the `back` prefix. 

With Logseq, for example, a word's page  has a references block, shows how many linked references of it. Each reference has an excerpt content, can be used as collecting pieces of thoughts about the word. In this way, it helps to organize your knowledge loosely, in other word, link your thinking.

Here is an example:

<img width="1601" alt="image" src="https://user-images.githubusercontent.com/25029451/167304120-3082fffa-115c-4052-8acc-ddf08e5ce889.png">

I'm not sure whether this is the so-called link map.

:)
